Assets/Scripts/Framework/Gameplay/LaneSystem/LaneManager.cs

```diff
@@ -52,7 +53,7 @@ private void RemoveNoteFromLane(HitStatus hitStatus, int laneID)
                 Destroy(target.gameObject);
             }
 
-            SessionValues.Instance.score += (int)hitStatus * SessionValues.Instance.scoreMultiplier;
+            SessionValues.Instance.score += (int)HitStatus.Perfect * SessionValues.Instance.scoreMultiplier;
```

Comment:
I think this change is a mistake.

Reply:
Whoops! Yeah that was left over from debugging! I will remove it ASAP.